test/acceptance/behave/components/common_steps/initial_steps.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+ Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+
+ This file is part of Orion Context Broker.
+
+ Orion Context Broker is free software: you can redistribute it and/or
+ modify it under the terms of the GNU Affero General Public License as
+ published by the Free Software Foundation, either version 3 of the
+ License, or (at your option) any later version.
+
+ Orion Context Broker is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+ General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+
+ For those usages not covered by this license please contact with
+ iot_support at tid dot es
+"""
+__author__ = 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+import behave
+from behave import step
+import logging
+
+from tools.properties_config import Properties    # methods in properties class
+from tools.fabric_utils import FabricSupport
+from tools.CB import CB
```

Comment:
I understand that the "common repo" is internal to TID and not public. It is not a good practise (from a open source community point of view) to make a public code depend on hiden code...